smrf/distribute/wind/winstral.py

```diff
@@ -107,6 +113,10 @@ def __init__(self, smrf_config):
 
         self.veg = v
 
+    @property
+    def thread_variables(self):
+        return list(self.BASE_THREAD_VARIABLES)
+
```

Comment:
Is this required? What breaks, when not overwriting and using the definition from the `image_data` class?

Reply:
Because I had in my mind that this didn't extend the `image_data` class, but it does...